server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -346,7 +346,7 @@ public void testSuccessfulSnapshotAndRestore() {
         assertNotNull(restoreSnapshotResponseListener.result());
         assertTrue(documentCountVerified.get());
         SnapshotsInProgress finalSnapshotsInProgress = masterNode.clusterService.state().custom(SnapshotsInProgress.TYPE);
-        assertFalse(finalSnapshotsInProgress.entries().stream().anyMatch(entry -> entry.state().completed() == false));
+        assertFalse(finalSnapshotsInProgress.forRepo(repoName).stream().anyMatch(entry -> entry.state().completed() == false));
```

Comment:
:facepalm:  Actually, this has always been a nonsensical assertion. Changed all spots to:

```
assertTrue(masterNode.clusterService.state().custom(SnapshotsInProgress.TYPE, SnapshotsInProgress.EMPTY).isEmpty());
```

... which is actually what this was supposed to assert (everything is done) instead of asserting that snapshots are either done or finalizing (which would be broken).